src/variables/utils/astim.ts

```diff
@@ -19,11 +19,19 @@ const parseAllVariables = (ast: File): MemberExpression[] => {
 }
 
 export const parseASTIM = (query: string): ASTIM => {
-  const ast: File = parse(query)
-  // Using `any` to circumvent ts error
-  const variables: any = ast ? parseAllVariables(ast) : []
+  let ast: File = null
+  try {
+    ast = parse(query)
+  } catch (_) {}
```

Comment:
can we console.error these instead of swallowing them? it helps support out later down the line